modules/oaiharvest/lib/oai_harvest_utils.py

```diff
@@ -118,60 +118,48 @@ def remove_duplicates(harvested_file_list):
     Usually happens when records are cross-listed across OAI sets.
 
     Saves a backup of original harvested file in: filename~
+
+    @param harvested_file_list: list of files to process and dedupe
     """
+    from copy import deepcopy
+    import lxml.etree as etree
```

Comment:
The same comment about lxml could apply here as well as in https://github.com/inveniosoftware/invenio/pull/2608#discussion_r22522110.  However for master we can raise the lxml requirement to mandatory very easily, if wanted.  (And we would not need to play with gentle fallbacks.)  What would you prefer?